docs/attributes-registry/event.md

```diff
@@ -9,7 +9,23 @@ linkTitle: Events
 <!-- semconv registry.event(omit_requirement_level) -->
 | Attribute  | Type | Description  | Examples  | Stability |
 |---|---|---|---|---|
-| `event.name` | string | Identifies the class / type of event. [1] | `browser.mouse.click`; `device.app.lifecycle` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `event.data` | string | The event payload serialized into JSON string. [1] | `{"role":"user","content":"how to use OTel Event API?"}`; `"plain string"` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
If this is aligned with the Body semantics of a LogRecord, why is it a `string` and not `Any`? Even in https://github.com/open-telemetry/semantic-conventions/pull/880 there is no strict requirement that it needs to be JSON.

Reply:
Attribute values within OTel will not support `Any` https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/common#attribute